src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -153,7 +153,7 @@ const ChatItemMyTextV2 = ({
   };
 
   return (
-    <View style={styles.chatContainer}>
+    <View style={[styles.chatContainer, isContinuous ? {marginTop: dimen.normalizeDimen(-4)} : {}]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional styling based on `isContinuous` is a good approach to handle dynamic styles. However, using negative margin can lead to overlapping elements and might cause unexpected behavior in some cases. Please ensure this doesn't cause any issues in your layout.
```diff
-    <View style={styles.chatContainer}>
+    <View style={[styles.chatContainer, isContinuous ? {marginTop: dimen.normalizeDimen(-4)} : {}]}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->